packages/rs-dpp/src/data_contract/document_type/class_methods/try_from_schema/v0/mod.rs

```diff
@@ -44,9 +44,9 @@ impl DocumentTypeV0 {
         data_contract_id: Identifier,
         name: &str,
         schema: Value,
-        schema_defs: Option<&BTreeMap<String, Value>>,
-        default_keeps_history: bool,
-        default_mutability: bool,
+        schema_defs: Option<&Value>,
+        default_revisions: bool,
+        default_read_only: bool,
```

Comment:
default_mutability

Reply:
Ok, I can bring mutability back and keep readOnly in JSON Schema since it is a special keyword for that.